hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -290,70 +287,77 @@ By default, no extra debug messages are enabled.
-rpi-eeprom-config pieeprom.bin > bootconf.txt                                                                          
-```
-Change `BOOT_ORDER` to be `0x21` instead of `0x1`. This tells the bootloader to try sd-card boot first and network boot second. You should normally include sd-card (`0x1`) in the boot sequence in-case of network failure.
+See: [Winbond W25x40cl datasheet](https://www.winbond.com/resource-files/w25x40cl_f%2020140325.pdf)
+
+* 1 - Configures the write protect regions to cover the entire EEPROM.
+* 0 - Clears write protect regions
+* -1 - Do nothing.
+
+Default: -1  
+Version: 2020-09-03   
+
+### bootloader_update
+This option may be set to 0 to block self-update without requiring the EEPROM configuration to be updated. This is sometimes useful when updating multiple Pis via network boot because this option can be controlled per Raspberry Pi (e.g. via a serial number filter in config.txt).
 
-### Apply the configuration change to the EEPROM image file
+Default: 1
+Version: 2020-09-03  
+
+## Advanced boot modes - Network / USB mass storage boot.
+For network or USB mass storage boot we recommend updating to bootloader version 2020-09-03 or newer.
+
+### Updating the bootloader
+#### Update using the Raspberry Pi Imager
+The easiest method to update the bootloader with a factory default configuration supporting USB boot is to use the Raspberry Pi Imager.
+1. Download the [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/)
+1. Download the latest [EEPROM recovery image](https://github.com/raspberrypi/rpi-eeprom/blob/master/releases.md)
+1. Select `Use Custom` to reformat and flash a **blank** SD card with the EEPROM update enabling USB MSD boot support.
+
+#### Manual bootloader update
+* From a Raspberry Pi OS 2020-08-20 or newer SD card:
 ```
-rpi-eeprom-config --out pieeprom-netboot.bin --config bootconf.txt pieeprom.bin
+sudo apt update
+sudo apt full-upgrade
 ```
-### Install the new EEPROM image
+* Run `vcgencmd bootloader_config` to check your current configuration and decide whether to use the factory default config or migrate your existing boot settings.
+* Update the bootloader
 ```
-sudo rpi-eeprom-update -d -f ./pieeprom-netboot.bin
-sudo reboot
+# Install with the factory default configuration (-d).
+sudo rpi-eeprom-update -a -d
 ```
-<a name="usbmassstorageboot"></a>
-## USB mass storage boot
+
+#### Changing the boot mode
+The easiest method to change the the default boot mode [raspi-config](../../configuration/raspi-config.md) to modify the `Boot Options`. Alternatively, edit the EEPROM configuration file to select the desired `BOOT_ORDER`.
+
+### USB mass storage boot
 USB MSD boot is now supported by the latest Raspberry Pi OS image (2020-08-20). After updating the bootloader using the following instructions a bootable USB drive may be created by using the [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) to flash the latest OS image to a USB MSD device.
```

Comment:
The instructions are now "preceding" rather than "following" :wink:

Reply:
Removed the sentence, already stated in the introduction to advanced boot modes.